torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19230,6 +19240,14 @@ def __init__(
         "_refs.reshape",
         torch_opinfo_name="reshape",
     ),
+    PythonRefInfo(
+        "_refs.rot90",
```

Comment:
This looks really good -- are the sample inputs for rot90 and/or roll sufficient, or should we use this opportunity to extend them with some reference inputs, too, to ensure all code paths and edge cases are validated? 

Also, although it is more work, error inputs for rot90 and roll would be great to verify their many error checks, but your call whether it's worth going there at this time